README.md

```diff
@@ -12,7 +12,7 @@ Giant Swarm offers a Kong Managed App which can be installed in workload cluster
 
 | Giant Swarm Chart Release | Upstream Chart Release | Kong Version | Kong IC Version | Kong-Gateway Enterprise container tag |
 | --- | --- | --- | --- | --- |
-| Unreleased | [2.33.0](https://github.com/Kong/charts/blob/main/charts/kong/CHANGELOG.md#2330) | [3.5.0](https://github.com/Kong/kong/blob/3.5.0/changelog/3.5.0/3.5.0.md) | [3.0.1](https://github.com/Kong/kubernetes-ingress-controller/blob/main/CHANGELOG.md#301) | 3.5.0.1-debian |
+| Unreleased | [2.33.1](https://github.com/Kong/charts/blob/main/charts/kong/CHANGELOG.md#2331) | [3.5.0](https://github.com/Kong/kong/blob/3.5.0/changelog/3.5.0/3.5.0.md) | [3.0.1](https://github.com/Kong/kubernetes-ingress-controller/blob/main/CHANGELOG.md#301) | 3.5.0.1-debian |
```

Comment:
Where is `Kong-Gateway Enterprise container tag` taken from?

Reply:
kong announces new versions in the gateway changelog: https://docs.konghq.com/gateway/changelog/#3501 
We then manually add the debian variant to retagger: https://github.com/giantswarm/retagger/blob/main/images/skopeo-docker-io.yaml#L73